meetings/osf_oauth2_adapter/migrations/0001_add_site_and_social_app.py

```diff
@@ -7,11 +7,11 @@
 from allauth.socialaccount.models import SocialApp
 from django.contrib.sites.models import Site
 
-from osf_oauth2_adapter.apps import OsfOauth2AdapterConfig
-from django.conf import settings
+from meetings.utils import OsfOauth2AdapterConfig
```

Comment:
So you couldn't import it from the eponymous app `osf_oauth2_adapter` dir into files app.py?